packages/teams-js/src/public/media.ts

```diff
@@ -412,30 +410,38 @@ export namespace media {
      * @param mediaEvent indicates what the event that needs to be signaled to the host client
      * Optional; @param callback is used to send app if host client has successfully handled the notification event or not
      */
-    protected notifyEventToHost(mediaEvent: MediaControllerEvent, callback?: (err?: SdkError) => void): void {
+    protected notifyEventToHost(mediaEvent: MediaControllerEvent, callback?: (err?: boolean) => void): void {
       ensureInitialized(FrameContexts.content, FrameContexts.task);
-      const err = isApiSupportedOnMobile(nonFullScreenVideoModeAPISupportVersion);
-      if (err) {
+      const isSupported = isMobileApiSupported(nonFullScreenVideoModeAPISupportVersion);
+      if (isSupported) {
         if (callback) {
-          callback(err);
+          return callback(isSupported);
         }
         return;
       }
 
       const params: MediaControllerParam = { mediaType: this.getMediaType(), mediaControllerEvent: mediaEvent };
-      sendMessageToParent('media.controller', [params], (err?: SdkError) => {
+
+      sendMessageToParent('media.controller', [params], (err?: boolean) => {
         if (callback) {
           callback(err);
         }
       });
     }
 
+    public stop(): Promise<void>;
```

Comment:
This version of `stop` needs documentation comments above it (I looked at `selectMedia` and if that was your example (since I suggested it as an example elsewhere ;)), it also needs documentation comments for it).

Reply:
I think adding comments to selectMedia might be a good example of something that is out of scope for Jess's PR. Totally agree that the function needs comments but at some point we need to stop pulling on the thread :)